lua/telescope/_extensions/packer.lua

```diff
@@ -65,23 +77,57 @@ local plugins = function(opts)
       end,
     },
     sorter = sorters.get_generic_fuzzy_sorter(),
-    attach_mappings = function(prompt_bufnr)
+    attach_mappings = function(prompt_bufnr, map)
       actions.select_default:replace(function()
-        local selection = action_state.get_selected_entry(prompt_bufnr)
+        local selection = action_state.get_selected_entry()
         actions.close(prompt_bufnr)
-		vim.cmd(string.format(":e %s", selection.readme))
-        -- packer[selection.value]()
+        vim.cmd(string.format(":e %s", selection.readme))
       end)
 
+      local open_online = function()
+        local selection = action_state.get_selected_entry()
+		actions._close(prompt_bufnr, true)
```

Comment:
only use `actions._close(prompt_bufnr, true)` if you go from picker to picker. Otherwise use `actions.close(prompt_bufnr)`

Reply:
If you do that you dont need `vim.cmd "stopinsert"` at the end

Reply:
Ok thanks!